include/seqan3/alphabet/aminoacid/translation.hpp

```diff
@@ -33,9 +33,9 @@ class rna15;
 /*!\brief Translate one nucleotide triplet into single amino acid (single nucleotide interface).
  * \ingroup aminoacid
  * \tparam nucl_type The type of input nucleotides.
- * \param[in] n1 First nucleotide in triplet.
- * \param[in] n2 Second nucleotide in triplet.
- * \param[in] n3 Third nucleotide in triplet.
+ * \param[in] n1 First seqan3::nucleotide_alphabet in triplet.
+ * \param[in] n2 Second seqan3::nucleotide_alphabet in triplet.
+ * \param[in] n3 Third seqan3::nucleotide_alphabet in triplet.
```

Comment:
I disagree with this change. We do not pass three different nucleotide alphabets, but rather three nucleotides to the function.

Reply:
I agree, I reversed this and added `must model seqan3::nucleotide_alphabet` to the tparam